lib/hexpm/repository/comment.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Hexpm.Repository.PackageReportComment do
+  use Hexpm.Schema
+  import Ecto.Query, only: [from: 2]
+
+  schema "comments" do
```

Comment:
The table name should match the module name. The migrations needs to change also.

```suggestion
  schema "package_report_comments" do
```